src/main/java/String07.java

```diff
@@ -0,0 +1,24 @@
+public class String07 {
+    public String decodeShortcutString(String str) {
+        String Characters = str.replaceAll("[0-9]", "");
+        String Nums = str.replaceAll("[^0-9]", ",");
```

Comment:
Tên biến không chuẩn, chú ý viết hoa thường
```suggestion
        String counts = str.replaceAll("[^0-9]", ",");
```

Reply:
Done